membersrvc/ca/validity_period_test.go

```diff
@@ -83,38 +83,38 @@ func TestValidityPeriod(t *testing.T) {
 	time.Sleep(time.Second * 240)
 
 	// 2. Obtain the validity period by querying and directly from the ledger
-	validityPeriod_A := queryValidityPeriod(t)
-	validityPeriodFromLedger_A := getValidityPeriodFromLedger(t)
+	validityPeriodA := queryValidityPeriod(t)
+	validityPeriodFromLedgerA := getValidityPeriodFromLedger(t)
 
 	// 3. Wait for the validity period to be updated...
 	time.Sleep(time.Second * 40)
 
 	// ... and read the values again
-	validityPeriod_B := queryValidityPeriod(t)
-	validityPeriodFromLedger_B := getValidityPeriodFromLedger(t)
+	validityPeriodB := queryValidityPeriod(t)
+	validityPeriodFromLedgerB := getValidityPeriodFromLedger(t)
 
 	// 5. Stop TCA and Openchain
 	stopServices(t)
 
 	// 6. Compare the values
-	if validityPeriod_A != validityPeriodFromLedger_A {
+	if validityPeriodA != validityPeriodFromLedgerA {
 		t.Logf("Validity period read from ledger must be equals tothe one obtained by querying the Openchain. Expected: %s, Actual: %s", validityPeriod_A, validityPeriodFromLedger_A)
```

Comment:
Build fails with this PR because the variable references in the log message have not been updated.